README.md

```diff
@@ -122,9 +122,9 @@ seen [here](governance/related-groups/README.md)
 
 ### Security TAG Chairs
 
-- Andrew Martin ([@sublimino](https://github.com/sublimino)), ControlPlane [Chair term: 3/17/2022 - 3/17/2024]
 - Pushkar Joglekar ([@PushkarJ](https://github.com/PushkarJ)), Independent [Chair term: 6/3/2023 - 6/3/2025]
 - Marina Moore ([@mnm678](https://github.com/mnm678)), NYU [Chair term: 10/3/2023 - 10/3/2025]
+- Eddie Knight ([@eddie-knight](https://github.com/eddie-knight)), Sonatype [Chair term: 7/5/2023 - 7/5/2025]
```

Comment:
Term dates seem wrong 😑

Reply:
US date format issues this side. Will close in favour of https://github.com/cncf/tag-security/pull/1251